cmd/completions.go

```diff
@@ -0,0 +1,81 @@
+// -----------------------------------------------------------------------------
+// SPDX-FileCopyrightText: Copyright © 2024 bomctl a Series of LF Projects, LLC
+// SPDX-FileName: cmd/completions.go
+// SPDX-FileType: SOURCE
+// SPDX-License-Identifier: Apache-2.0
+// -----------------------------------------------------------------------------
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+package cmd
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/protobom/protobom/pkg/sbom"
+	"github.com/spf13/cobra"
+
+	"github.com/bomctl/bomctl/internal/pkg/db"
+	"github.com/bomctl/bomctl/internal/pkg/sliceutil"
+)
+
+func completions(
+	cmd *cobra.Command,
+	_ []string,
+	toComplete string,
+) ([]string, cobra.ShellCompDirective) {
+	backend := backendFromContext(cmd)
+
+	comps := []string{}
+
+	defer backend.CloseClient()
+
+	documents, err := backend.GetDocumentsByIDOrAlias()
+	if err != nil {
+		backend.Logger.Fatal(err)
```

Comment:
Should probably just return the expected values rather than hard failing

```suggestion
		return nil, cobra.ShellCompDirectiveError
```